### DISCUSSION TITLE
Email (Update top issues dashboard: All jobs have failed)

### DISCUSSION THREAD

**Discussion by Sachin9328** [anuraghazra/github-readme-stats#2505] [Q&A] [answered]
I keep getting an email saying "Run failed: Update top issues dashboard", I literally have no idea what it means, I am just using github-readme-stats for my profile readme and I have no idea about any of the advanced stuff, can someone tell me what it means in simple terms and how do I stop getting this email.

**Comment by mkredzel** [comment, marked as answer]
Hi @Sachin9328,

It looks like you must have subscribed to GitHub Actions (CI activity) for this repo.

Could you please double check your "watch" setting? You should also be able to unsubscribe via [notification page](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/managing-your-subscriptions).

I hope this helps.

<img width="313" alt="Screenshot 2023-02-08 at 16 44 37" src="https://user-images.githubusercontent.com/44983911/217595392-4e83ebd0-00a7-4ca4-8d7f-158e4741a3da.png">

**Reply by Zo-Bro-23** [reply]
Thanks for the quick answer!
